refresh/templates/common/Package.swift

```diff
@@ -11,6 +11,7 @@ let package = Package(
     dependencies: [
         .Package(url: "https://github.com/IBM-Swift/Kitura.git",             majorVersion: 1, minor: 7),
         .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git",       majorVersion: 1, minor: 7),
+        .Package(url: "https://github.com/IBM-Swift/Health.git",             majorVersion: 0),
```

Comment:
It would be better if the Health package were at v1 before we started to rely on it (http://semver.org/#how-do-i-know-when-to-release-100)